### PR TITLE
Translate inspecting html and css

### PR DESCRIPTION
**Pull request by emredevsalot**

### CODE REVIEW

File: temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
+Stiller bölmesi, stilleri doğrudan tarayıcıda değiştirmenize de olanak tanır. Yeni bir kural eklemek için herhangi bir seçicinin içine tıklayabilir veya mevcut bir nitelik veya değere tıklayarak değişiklik yapabilirsiniz. Bunu yaptığınızda, internet sayfası gerçek zamanlı olarak değişikliklerle yanıt verir. Bu değişiklikler, metin düzenleyicinizdeki kaynak kodunu etkilemez, ancak sayfayı tekrar tekrar yüklemeye gerek kalmadan çeşitli nitelikleri ve değerleri hızlı bir şekilde test edebilmek için son derece yararlıdır.
 
-### Assignment
+### Ödev
 
 <div class="lesson-content__panel" markdown="1">
-Go through the following sections of the [official Chrome DevTools docs](https://developers.google.com/web/tools/chrome-devtools):
+[Resmi Chrome Geliştiri Araçları dökümanının](https://developers.google.com/web/tools/chrome-devtools) aşağıdaki bölümlerini inceleyin:
```
Comment (sevilayerkan) [inline temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md]: There is a small mistype in `Geliştiri` it should be `Geliştirici`

File: temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
 
-- [Overview](https://developer.chrome.com/docs/devtools/overview/): don't navigate to any other pages linked here; just get familiar with _what_ tools are available in the DevTools, rather than how to use all of them right now.
-- [Open Chrome DevTools](https://developer.chrome.com/docs/devtools/open/): similar to what we went over above, but with some nice extras.
-- [View and change CSS](https://developer.chrome.com/docs/devtools/css): be sure to follow along with any interactive instructions!
-- [Get Started With Viewing And Changing The DOM](https://developer.chrome.com/docs/devtools/dom/): skip through any part that uses the JavaScript console.
+- [Overview](https://developer.chrome.com/docs/devtools/overview/): burada bağlantısı verilen diğer sayfalara gitmeyin; Şimdi hepsini nasıl kullanacağınızdan ziyade DevTools'ta _hangi_ araçların mevcut olduğunu öğrenin.
```
Comment (sevilayerkan) [inline temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md]: Word that comes after `;` should start with a lowercase letter.

File: temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+### Giriş
+
+HTML ve CSS'inizi inceleyebilmek ve hata ayıklayabilmek, önyüz geliştirme için çok önemlidir. Bu derste, öğeleriniz ve CSS kurallarınız hakkında ayrıntılı bilgileri görmenizi sağlayan ve kodunuzdaki sorunları bulup düzeltmenize yardımcı olan Chrome Geliştirme Araçlarını inceleyeceğiz.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
```
Comment (cansirin) [inline temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md]:
```suggestion
Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.
```

File: temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+- [Tarayıcınızın geliştirici araçlarıyla sayfadaki belirli bir öğeyi nasıl seçersiniz?](#inspecting-elements)
+- [Bir CSS ifadesindeki üstü çizili kısım, tarayıcınızın geliştirici araçlarında ne anlama gelir?](#strikethrough)
+- [Tarayıcınızın geliştirici araçlarıyla sayfadaki belirli öğelerinde CSS'i gerçek zamanlı olarak nasıl değiştirirsiniz?](#testing-styles-in-the-inspector)
+
+### Ek Kaynaklar
+
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
+
+- [This article about how we can utilize css overview in the developer tools](https://www.freecodecamp.org/news/how-to-use-css-overview-in-chrome-developer-tools/) to check the colors, font styles, media-queries, etc. used on a particular webpage.
```
Comment (cansirin) [inline temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md]: Don't we need translation for here?

File: temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Şu anda karşınıza çıkan tüm araçlar sizi endişelendirmesin! Bu ders için yalnızca Öğeler ve Stiller bölmelerine odaklanacağız.
+
+### Öğeleri Denetlemek
+
+Öğeler bölmesinde, sayfanın tüm HTML yapısını görebilirsiniz. Belirli bir öğeyi seçmek için bu bölmede herhangi birine tıklayabilirsiniz. Alternatif olarak, aşağıdaki görselin sol üstünde bulunan mavi simgeye tıklayıp sayfadaki herhangi bir öğenin üzerine gelebilirsiniz.
+
+![Inspector Icon](https://cdn.statically.io/gh/TheOdinProject/curriculum/594984d7c9f9e744577f19ea475b3864e8cc7c91/html_css/v2/foundations/inspecting-html-and-css/imgs/01.png)
+
+<span id="strikethrough">Bir öğe seçildiğinde, Stiller sekmesi, uygulanmakta olan tüm stillerin yanı sıra üzerine yazılan stilleri de gösterir (metnin üstü çizili olarak gösterilir).</span> Örneğin, ana sayfadaki "Web Geliştirme Kariyeriniz Burada Başlıyor" başlığına tıklamak için denetçiyi kullanırsanız, aşağıda görüldüğü gibi, sağ tarafta şu anda öğeyi etkileyen tüm stilleri göreceksiniz.
```
Comment (cansirin) [inline temel_bilgiler/html_css/html-ve-css-inceleme/html-ve-css-inceleme.md]:
```suggestion
<span id="strikethrough">Bir öğe seçildiğinde, Stiller bölmesi, uygulanmakta olan tüm stillerin yanı sıra üzerine yazılan stilleri de gösterir (metnin üstü çizili olarak gösterilir).</span> Örneğin, ana sayfadaki "Web Geliştirme Kariyeriniz Burada Başlıyor" başlığına tıklamak için denetçiyi kullanırsanız, aşağıda görüldüğü gibi, sağ tarafta şu anda öğeyi etkileyen tüm stilleri göreceksiniz.
```

**Review by cansirin — CHANGES_REQUESTED** [review]

**Pull request merged**